### DISCUSSION TITLE
Issues reading 48bit Tiff, bug or user error

### DISCUSSION THREAD

**Discussion by ehardestyOld** [SixLabors/ImageSharp#1715] [Q&A]
I am using SixLabors.Imagesharp 1.0.4-alpha.0.50 to work with a 48 bit Tiff image that came from a scanner. However, I am only getting a garbled imaged

Source image looks like this: 
![image](https://user-images.githubusercontent.com/70970604/127186290-55754d24-cc90-42ee-8e3b-9cd39d54bd5f.png)

For debug purposes I stripped the code down to simply loading the image and saving it back to disk as a PNG
```
using (var image = await Image.LoadAsync(file))
{
    await image.SaveAsPngAsync(_debugOutput + fileName + ".png");
}
```

Output image:
![image](https://user-images.githubusercontent.com/70970604/127186674-3787e7b2-ef05-47cf-90dc-41cbfc6d1e3d.png)

Note that the black rectangle was added to anonymize the image but the original pre-edit has the same result.

Am I doing something wrong here? (wouldn't be surprising) If not, should I create an issue?

[TiffIssue.zip](https://github.com/SixLabors/ImageSharp/files/6886984/TiffIssue.zip)

**Comment by brianpopow** [comment]
I can reproduce it and created an [Issue](https://github.com/SixLabors/ImageSharp/issues/1716)

@ehardesty thanks for the report.

**Comment by brianpopow** [comment]
@ehardesty with #1719 merged, this issue is fixed. You can try our Nighty builds from the [MyGet Feed](https://www.myget.org/feed/sixlabors/package/nuget/SixLabors.ImageSharp)
